repos/fdbt-site/cypress_tests/cypress/integration/myFares/myFares.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    addFlatFareProductIfNotPresent,
+    clickElementById,
+    clickElementByText,
+    clickRandomElementsInTable,
+    getHomePage,
+} from '../../support/helpers';
+
+describe('Able to access services in my fares', () => {
+    it('Able to access service pages', () => {
+        getHomePage();
+        clickElementById('manage-fares-link');
+        clickRandomElementsInTable('govuk-table__body', 'service-link');
+    });
+    it('Able to access service pages via operator settings', () => {
+        getHomePage();
+        clickElementById('account-link');
+        clickElementByText('Services');
+        clickRandomElementsInTable('govuk-table__body', 'service-link');
+    });
+});
+
+describe('Able to access other products in my fares', () => {
+    before(() => {
+        addFlatFareProductIfNotPresent();
+    });
+    it('Able to access other products pages', () => {
```

Comment:
Could we change the name of the test to flow on from the word 'it' ?

Reply:
Good shout, done that 👍